kolibri/core/assets/src/api-resource.js

```diff
@@ -187,8 +190,12 @@ class Model {
     return promise;
   }
 
+  get orderedUrlParams() {
+    return this.resource.urlParams.map((key) => this.urlParams[key]);
+  }
```

Comment:
`urlParams` on the resource is an Array, in order, of the kwargs that fill in the URL.